packages/relayer-lib/src/tx_builder/cosmos_to_cosmos.rs

```diff
@@ -217,4 +217,39 @@ impl TxBuilderService<CosmosSdk, CosmosSdk> for TxBuilder {
         }
         .encode_to_vec())
     }
+
+    #[tracing::instrument(skip_all)]
+    async fn update_client(&self, dst_client_id: String) -> Result<Vec<u8>> {
```

Comment:
We should add some more logging here, similar to what we do in relay_events to make it easier to debug in deployment

Reply:
added